tensorflow/compiler/jit/xla_device_ops.h

```diff
@@ -28,6 +28,7 @@ limitations under the License.
 #include "tensorflow/core/kernels/data/prefetch_dataset_op.h"
 #include "tensorflow/core/kernels/fifo_queue.h"
 #include "tensorflow/core/kernels/function_ops.h"
+#include "tensorflow/core/kernels/functional_ops.h"
```

Comment:
Could you also update BUILD file?
`./tensorflow/compiler/jit/xla_device_ops.h:31:10: fatal error: 'tensorflow/core/kernels/functional_ops.h' file not found`